cmd/humanlog/main_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestApplyConfigFromConfigFile_when_one_of_skip_or_keep_is_given(t *testing.T) {
+
+	wd, _ := os.Getwd()
+	dirs := strings.Split(wd, "/")
```

Comment:
fyi: when manipulating filepath, use https://pkg.go.dev/path/filepath since it's cross platform